terraform/modules/concourse_web/iam.tf

```diff
@@ -32,20 +32,55 @@ data "aws_iam_policy_document" "web" {
   }
 }
 
-resource "aws_iam_role_policy" "parameter_store" {
-  name   = "${local.name}ParameterStoreAccess"
-  role   = aws_iam_role.web.id
-  policy = data.aws_iam_policy_document.secrets.json
+data "aws_iam_policy_document" "concourse_secretsmanager" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "secretsmanager:ListSecrets",
+      "secretsmanager:GetSecretValue",
+      "secretsmanager:DescribeSecret"
+    ]
+
+    resources = [
+      "arn:aws:secretsmanager:::secret:/concourse/*",
```

Comment:
are the secrets manually created and populated?

Reply:
They are at the moment. There's another ticket in the backlog to put a workflow around secrets held in secret manager. At that point this ARN can be changed to reference that TF resource.

Reply:
We've also left SSM access in.  We need to test if we can lift and shift pipelines and migrate to SM after, or if we need to migrate to SM right away.